website/content/en/docs/upgrading/compatibility.md

```diff
@@ -86,6 +86,16 @@ By adopting this practice we allow our users who are early adopters to test out
 
 ### Snapshot Releases
 
-We release a snapshot release for every commit that gets merged into the main repository. This enables our users to immediately try a new feature or fix right after it's merged rather than waiting days or weeks for release.
-Snapshot releases are suitable for testing, and troubleshooting but users should exercise great care if they decide to use them in production environments.
-Snapshot releases are tagged with the git commit hash prefixed by the Karpenter major version. For example `v0-fc17bfc89ebb30a3b102a86012b3e3992ec08adf`. For more detailed examples on how to use snapshot releases look under "Usage" in [Karpenter Helm Chart](https://gallery.ecr.aws/karpenter/karpenter).
+We release a snapshot release for every commit that gets merged into the [`aws/karpenter-provider-aws`](https://www.github.com/aws/karpenter-provider-aws). This enables our users to immediately try a new feature or fix right after it's merged rather than waiting days or weeks for release.
```

Comment:
```suggestion
We release a snapshot release for every commit that gets merged into [`aws/karpenter-provider-aws`](https://www.github.com/aws/karpenter-provider-aws). This enables users to immediately try a new feature or fix right after it's merged rather than waiting days or weeks for release.
```